circuits/cpp/src/aztec3/circuits/kernel/private/utils.cpp

```diff
@@ -91,7 +91,7 @@ std::shared_ptr<NT::VK> fake_vk()
 {
     std::map<std::string, NT::bn254_point> commitments;
     commitments["FAKE"] = *new NT::bn254_point(NT::fq(0), NT::fq(0));
-    NT::VKData vk_data = { .circuit_type = static_cast<uint32_t>(proof_system::CircuitType::TURBO),
+    NT::VKData vk_data = { .circuit_type = static_cast<uint32_t>(1),
```

Comment:
can we use an enum like before instead of just 1?

Reply:
yeah just ignore this for now, I'm just mucking around trying to extract turbo from circuits code